### PR TITLE
MI-PYT: Be consistent with __venv__

### PR DESCRIPTION
**Pull request by hroncok**

### CODE REVIEW

**Comment by hroncok** [comment]
cc @MarekSuchanek

File: lessons/intro/docs/index.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ potvrdit (<kbd>Enter</kbd>).
 K modulům `autodoc` a `doctest` se dostaneme později.
 
 ```ansi
-␛[36m$␛[0m . env/bin/activate
-␛[36m(env)$␛[0m python -m pip install sphinx
-␛[36m(env)$␛[0m python -m sphinx.quickstart
+␛[36m$␛[0m . __env__/bin/activate
```
Comment (encukou) [inline lessons/intro/docs/index.md]: This isn't consistent :)

Comment (hroncok) [reply lessons/intro/docs/index.md]: Oh, right.

**Comment by hroncok** [comment]
Amended.

**Comment by encukou** [comment]
Should [intro/distribution](http://naucse.python.cz/course/mi-pyt/intro/distribution/) be changed as well?

**Comment by hroncok** [comment]
Should. But Flask has to be changed soon. Others may be changed later.

**Comment by encukou** [comment]
I see.
LGTM, merge at will.

On Oct 9, 2017 5:37 PM, "Miro Hrončok" <notifications@github.com> wrote:

Should. But Flask has to be changed soon. Others may be changed later.

—
You are receiving this because your review was requested.
Reply to this email directly, view it on GitHub
<https://github.com/pyvec/naucse.python.cz/pull/215#issuecomment-335195242>,
or mute the thread

**Pull request merged**